README.md

```diff
@@ -1,24 +1,35 @@
+***sudo-gcp is currently in alpha stages! Expect breaking changes.***
+
 # Sudo GCP
 
 This tool helps with running Google Cloud commands with temporary elevated
-privileges.
+privileges using short-lived OAuth access tokens.
+
+Access tokens are stored in the OS's keychain/secret store and reused if they have 
+matching scopes and are not expired for subsequent commands.
```

Comment:
```suggestion
```suggestion
`sudo-gcp` securely caches access tokens using the operating system's secret-store/keychain, and
will reuse matching non-expired tokens on subsequent calls. 
```